### ISSUE TITLE
Statement of Purpose

### ISSUE THREAD

**Issue by ylluminate** [orbitalimpact/opal-phaser#5]
Hey guys, wanted to rip out a quick statement of purpose and goals in order to get some feedback from @ylluminarious @gabrielrios @andreanastacio and anyone else that would like to chime in.  We can then leverage this as a guiding doc to flesh out many and various bitesized tasks for going forward towards getting this to 1.0.

I know that we have internally discussed the goal for 1.0 of this support wrapper being to have full support for the current Phaser release that is close or near to the time when we are roughly ready to release.  The idea is that we have full coverage of Phaser such that there is complete Ruby centric (where rubyists will feel right at home) access to the Phaser API in place for Ruby developers.

This is an iterative process that will be implemented by 1) implementing all Phaser examples in Ruby (Opal) and 2) developing some of our own game(s) along the way to keep things interesting.

Any suggestions or input on how to make this more actionable with bitesized tasks (issues) here would be fantastic and appreciated guys!  We can chunk this out into individual trackable issues as we get some round table discussion going.

**Comment by ylluminarious** [comment]
Yeah, I do think that a lot of the API needs to be changed around a bit to feel more Rubyist. I've been seeing that a lot as I've been replicating these Phaser Examples. Anything that anyone can do to make the API more Rubyist is certainly welcome. 
I also currently have a Phaser game that I've been working on for a while that I'm planning to rewrite in opal-phaser, and hopefully use that game to market opal-phaser and make it have a unique game of its own.
